src/components/NavForTab/NavForTab.jsx

```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "react-router-dom";
+import "./NavForTab.css";
+export const NavForTab = () => {
+  return (
+    <div className="nav-bar-tab">
+      <div className="nav-bar-tab-container">
+        <NavLink to="/">
+          <img
+            className="nav-bar-tab--icon"
+            src="/assets/image/HomePage-Images/home.png"
+            alt="Home"
+          />
+        </NavLink>
+        <NavLink to="/report/">
+          <img
+            className="nav-bar-tab--icon"
+            src="/assets/image/HomePage-Images/Report.png"
+            alt="Report"
+          />
+        </NavLink>
+        <NavLink to="/login/">
+          <img
+            className="nav-bar-tab--icon"
+            src="/assets/image/HomePage-Images/Login.png"
+            alt="Login"
+          />
+        </NavLink>
```

Comment:
`NavLink` is use for style active links so if you do not want to use active style then use `<Link>`

Reply:
updated,Thankyou